Fix Unknown AMIs and regions in docs (#1009)
berops/claudie: docs/input-manifest/providers/aws.md

1. Install AWS CLI tools by following [this guide](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
2. Setup AWS CLI on your machine by following [this guide](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html).

> 3. Ensure that the necessary regions are present in your AWS account. You can check the available regions using this guide [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions). If some regions are missing, you may encounter an error suggesting your STS token is invalid.

Reviewer: ```suggestion
3. Ensure that the regions you're planning to use are enabled in your AWS account. You can check the available regions using [this guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions), and you can enable them using [this guide](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-regions.html). Otherwise, you may encounter a misleading error suggesting your STS token is invalid.
```

Nitpick, possibly better wording

        serverType: t3.medium
        # AMI ID of the image.
        # Make sure to update it according to the region. 

>         image: ami-0618becf24e8d2b5f

Reviewer: ```suggestion
        image: ami-0e4d1886bf4bb88d5
```

        storageDiskSize: 50

      - name: compute-2-aws
        providerSpec:
          # Name of the provider instance.
          name: aws-1
          # Region of the nodepool.

>           region: eu-central-3 # this is just an example, it does not really exists!

Reviewer: ```suggestion
          region: eu-central-2
```

Reviewer: Could we just change this to a region that exists, then? What do you think @Despire ?

Reviewer 2: Yes, this was probably a typo in the docs

Reviewer: I'll commit this suggestion and resolve conversation then

          # Availability zone of the nodepool.

>           zone: eu-central-3a 

Reviewer: ```suggestion
          zone: eu-central-2a
```

        serverType: t3.medium
        # AMI ID of the image.
        # Make sure to update it according to the region. 

>         image: ami-0618becf24e8d2b5f

Reviewer: ```suggestion
        image: ami-0e4d1886bf4bb88d5
```
